pkg/pool-utils/test/ExternalFees.test.ts

```diff
@@ -34,7 +34,7 @@ describe('ExternalFees', function () {
         expect(await mock.bptForPoolOwnershipPercentage(1, FP_100_PCT.sub(1))).to.be.eq(fp(1).sub(1));
 
         expect(await mock.bptForPoolOwnershipPercentage(fp(1), fp(0.5))).to.be.eq(fp(1));
-        expect(await mock.bptForPoolOwnershipPercentage(fp(1), fp(0.25))).to.be.almostEqual(fp(0.333333333333333333));
+        expect(await mock.bptForPoolOwnershipPercentage(fp(1), fp(0.25))).to.be.almostEqual(fp(0.3333333333333333));
```

Comment:
Was this an automated change? This seems odd.

Maybe to do with JS float precision?

Reply:
![image](https://user-images.githubusercontent.com/15848336/200552979-6bcfdb44-c5b5-40b3-8f51-5a0d1a2fccbd.png)
Looks like it is. JS seems to trim this value automatically.

Reply:
Yes, the linter issued the precision error with the previous amount of decimals, so I trimmed it down until the error was gone.

The result shouldn't be affected by this change